packages/app-store/office365calendar/api/callback.ts

```diff
@@ -74,5 +75,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   });
 
   const state = decodeOAuthState(req);
-  return res.redirect(getSafeRedirectUrl(state?.returnTo) ?? "/apps/installed");
+  return res.redirect(
+    getSafeRedirectUrl(state?.returnTo) ??
+      getInstalledAppPath({ variant: "calendar", slug: "office365-calendar" })
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.